backend/app/services/rag/index/indexer.py

```
@@ -8,9 +8,9 @@
 
 from datetime import datetime, timezone
 from pathlib import Path
-from typing import Dict, Optional, Union
+from typing import Dict, Optional, Union, List, Optional
```

Ruff (0.14.10)

Redefinition of unused `Optional` from line 11

Remove definition: `Optional`

(F811)